Gemfile.lock

```diff
@@ -293,15 +297,15 @@ GEM
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2021.0225)
-    mimemagic (0.4.3)
+    mimemagic (0.3.10)
```

Comment:
Why did mimemagic get downgraded? Is this necessary to have the feature function?

Reply:
Never mind, this change was motivated by a change in the main branch in order to mitigate security warning from Hakiri.